server/sentry.js

```diff
@@ -241,11 +241,26 @@ Sentry.prototype._refresh = function () {
 Sentry.prototype._checkSentries = function () {
   var interval = Math.floor(this._checkInterval)
 
-  logging.info('#presence - #sentry checking sentries:', this.name)
-  this._loadAndCleanUpSentries()
+  if (!this.isTilted()) {
+    logging.info('#presence - #sentry checking sentries:', this.name)
+    this._loadAndCleanUpSentries()
+  }
+
+  this._lastChecked = Date.now()
   this._checkSentriesTimer = setTimeout(this._checkSentries.bind(this), interval)
 }
 
+// tilt mode: based on http://redis.io/topics/sentinel#tilt-mode
+// before recognizing a remote sentry as down,
+// make sure we trust our own observations by checking for clock skew.
```

Comment:
I love this: "make sure we trust our own observations by checking for clock skew"